src/app/tests/suites/certification/Test_TC_LVL_2_1.yaml

```diff
@@ -23,116 +23,140 @@ tests:
+    - label: "Reads the CurrentFrequency attribute"
       command: "readAttribute"
-      attribute: "On Off Transition Time"
+      attribute: "current frequency"
       response:
           value: 0
+          constraints:
+              type: uint16
 
-    - label: "sends a Move to level command"
-      command: "MoveToLevel"
-      arguments:
-          values:
-              - name: "level"
-                value: 254
-              - name: "transitionTime"
-                value: 65535
-              - name: "optionMask"
-                value: 1
-              - name: "optionOverride"
-                value: 1
-
-    - label: "Wait 10ms"
-      cluster: "DelayCommands"
-      command: "WaitForMs"
-      arguments:
-          values:
-              - name: "ms"
-                value: 10
-
-    - label: "reads current Level attribute from DUT"
-      command: "readAttribute"
-      attribute: "current Level"
-      response:
-          value: 254
-
-    - label: "Reset level to 254"
-      command: "MoveToLevel"
-      arguments:
-          values:
-              - name: "level"
-                value: 254
-              - name: "transitionTime"
-                value: 0
-              - name: "optionMask"
-                value: 1
-              - name: "optionOverride"
-                value: 1
-
-    - label: "Wait 100ms"
-      cluster: "DelayCommands"
-      command: "WaitForMs"
-      arguments:
-          values:
-              - name: "ms"
-                value: 100
+    - label: "Reads the MinFrequency attribute"
+      command: "readAttribute"
+      attribute: "min frequency"
+      response:
+          value: 0
+          constraints:
+              type: uint16
+
+    - label: "Reads the MaxFrequency attribute"
+      command: "readAttribute"
+      attribute: "max frequency"
+      response:
+          value: 0
+          constraints:
+              type: uint16
+
+    - label: "Reads the OnOffTransitionTime attribute"
+      command: "readAttribute"
+      attribute: "on off transition time"
+      response:
+          value: 0
+          constraints:
+              type: uint16
+
+    #Disabled due to issue - #11670
+    - label: "Reads the OnLevel attribute "
+      disabled: true
+      command: "readAttribute"
+      attribute: "on level"
+      response:
+          value: null
+
+    - label: "Reads the OnLevel attribute "
+      command: "readAttribute"
+      attribute: "on level"
+      response:
+          constraints:
+              type: uint8
+
+    #Disabled due to issue - #11670
+    - label: "Reads the OnTransitionTime attribute "
+      disabled: true
+      command: "readAttribute"
+      attribute: "on transition time"
+      response:
+          value: null
+
+    - label: "Reads the OnTransitionTime attribute "
+      command: "readAttribute"
+      attribute: "on transition time"
+      response:
+          constraints:
+              type: uint16
+
+    #Disabled due to issue - #11670
```

Comment:
Again, this seems like it needs a separate issue.  This applies in a bunch of other places in this PR too.

Reply:
i will update with proper issue ID